aws/logs_monitoring/setup.py

```diff
@@ -24,4 +24,5 @@
     extras_require={
         "dev": ["nose2==0.9.1", "flake8==3.7.9", "requests==2.22.0", "boto3==1.10.33"]
     },
+    py_modules=[],
```

Comment:
this is needed because of https://github.com/pypa/setuptools/issues/3197#issuecomment-1078770109